FirebaseStorage/Sources/StorageReference.swift

```diff
@@ -410,15 +410,15 @@ import Foundation
    *       the current `StorageReference`.
    */
   @objc(listAllWithCompletion:)
-  open func listAll(completion: @escaping ((_: StorageListResult?, _: NSError?) -> Void)) {
+  open func listAll(completion: @escaping ((_: StorageListResult?, _: Error?) -> Void)) {
```

Comment:
I think to have caught this, we would have needed explicit typing in the API tests: 

https://github.com/firebase/firebase-ios-sdk/blob/3d51280713332af3f023abc7e305541871c1e005/FirebaseStorage/Tests/Unit/StorageAPITests.swift#L86-L87

```swift
ref.listAll { listResult: StorageListResult?, error: Error? in 
 } 
```

Reply:
We'd have to do this for all Swift closures to get complete coverage.

Reply:
Good point! Done for the StorageAPI tests in this PR.